catalog/test.js

```diff
@@ -1 +0,0 @@
-console.log(a === undefined)
```

Comment:
i'm not sure where this came from 🤔 
@fiskus any idea?

Reply:
Probably, I tested eslint or husky, and missed that it was commited